geode-core/src/test/java/org/apache/geode/management/internal/cli/commands/AlterAsyncEventQueueCommandDUnitTest.java

```diff
@@ -28,10 +28,11 @@
 import org.apache.geode.test.dunit.rules.ClusterStartupRule;
 import org.apache.geode.test.dunit.rules.MemberVM;
 import org.apache.geode.test.junit.categories.DistributedTest;
+import org.apache.geode.test.junit.categories.GfshTest;
 import org.apache.geode.test.junit.rules.GfshCommandRule;
 
 
-@Category(DistributedTest.class)
+@Category({DistributedTest.class, GfshTest.class})
```

Comment:
We could consider tagging command tests with GfshCommandTest instead of GfshTest and then reserve GfshTest for Gfsh internals. For example, this test would end up with:
```java
@Category({DistributedTest.class, GfshCommandTest.class, AsyncEventQueueTest.class})
```
And then this test could be maintained by anyone wanting to maintain AsyncEventQueue.

Reply:
If I understand the original intent of test categorization, The test categories should match JIRA components. `GfshCommand` is not a component, so unless we add that to JIRA we should not create a `GfshCommandTest` category. 

How about reserving `GfshTest` for Gfsh internals as @kirklund suggested. Any tests of Gfsh commands should only include the category for the command's functionality. @kirklund example would just be 
```@Category({DistributedTest.class, AsyncEventQueueTest.class})```

Reply:
I generally like that idea, but am not sure how the GfshCommandTest category would be used. 100% of what's in it would also be part of another test category. Maybe we can revisit the gfsh command related tests once we are done with the basic initial categorization?